ttm/cli.py

```diff
@@ -68,3 +68,15 @@ def do_run(self, subcmd, opts, project):
         if ToTestPublisher(self.tool).publish(project) == QAResult.passed:
             ToTestPublisher(self.tool).wait_for_published(project)
         ToTestReleaser(self.tool).release(project)
+
+    @cmdln.option('--status', choices=['testing', 'publishing'], help='The status to fix')
```

Comment:
It's something that should be used manually only, so might as well allow the other choices as well here?

Reply:
Yeah, but the others don't really have any mechanism value, so I don't consider them important enough. If I listed more here they would run out of sync just too quickly.

But I can drop the choices and make it a string argument - it's for experts only anyway

Reply:
I moved the choices to help text now